src/main/java/nextstep/subway/auth/application/LoginCheckService.java

```diff
@@ -0,0 +1,13 @@
+package nextstep.subway.auth.application;
+
+import nextstep.subway.auth.domain.LoginMember;
+import org.springframework.stereotype.Service;
+
+@Service
+public class LoginCheckService {
+    public void needLogin(LoginMember loginMember) {
+        if(loginMember.isGuest()) {
```

Comment:
컨벤션도 살짝 다시한번 리마인드 하시죠 😄 
```suggestion
        if (loginMember.isGuest()) {
```

Reply:
ㄷㄷ 이걸 깜빡했네요